index.php

```diff
@@ -4,6 +4,16 @@
 include('-/SIDB423.php');
 include('-/db.php');
 
+define('LESSN_VERSION',	'1.1.1');
+
+define( 'LESSN_ROOT', __DIR__ );
+define('LESSN_DOMAIN', 	preg_replace('#^www\.#', '', $_SERVER['SERVER_NAME']));
+define('LESSN_URL', 	str_replace('-/index.php', '', 'http://'.LESSN_DOMAIN.$_SERVER['PHP_SELF']));
+
+define('COOKIE_NAME', 	DB_PREFIX.'auth');
+define('COOKIE_VALUE',	md5(USERNAME.PASSWORD.COOKIE_SALT));
+define('COOKIE_DOMAIN', '.'.LESSN_DOMAIN);
```

Comment:
The addition of all of these constants is curious; how does this help with the list view? Is it truly necessary, or something that ended up in here that is related, but separate?